src/databricks/labs/remorph/deployment/job.py

```diff
@@ -100,15 +101,16 @@ def _recon_job_settings(
                         job_cluster_key="Remorph_Reconciliation_Cluster",
                     ),
                     recon_config,
+                    remorph_wheel_path,
                 ),
             ],
             "max_concurrent_runs": 2,
             "parameters": [JobParameterDefinition(name="operation_name", default="reconcile")],
         }
 
-    def _job_recon_task(self, jobs_task: Task, recon_config: ReconcileConfig) -> Task:
+    def _job_recon_task(self, jobs_task: Task, recon_config: ReconcileConfig, remorph_wheel_path: str) -> Task:
         libraries = [
-            compute.Library(pypi=compute.PythonPyPiLibrary("databricks-labs-remorph")),
+            compute.Library(whl=f"{remorph_wheel_path}"),
```

Comment:
```suggestion
            compute.Library(whl=remorph_wheel_path),
```
why do you need to wrap one variable into a string?

Reply:
Done